back/node_watcher/src/tasks/createReferendumStatus.ts

```diff
@@ -63,7 +63,7 @@ const createReferendumStatus: Task<NomidotReferendumStatusUpdate[]> = {
         }
 
         const relatedReferendum = await prisma.referendum({
-          id: parseInt(referendumRawEvent.ReferendumIndex),
+          referendumId: parseInt(referendumRawEvent.ReferendumIndex),
```

Comment:
Would it be possible to do `.toJSON()` [here](https://github.com/paritytech/Nomidot/blob/master/back/node_watcher/src/tasks/createReferendum.ts#L41), so that you would'nt need a parseInt

Reply:
sure, let me try that.